commands/team_users.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package commands
+
+import (
+	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mmctl/client"
+	"github.com/mattermost/mmctl/printer"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var TeamUsersCmd = &cobra.Command{
+	Use:   "users",
+	Short: "Management of team users",
+}
+
+var TeamUsersRemoveCmd = &cobra.Command{
+	Use:        "remove [team] [users]",
+	Short:      "Remove users from team",
+	Long:       "Remove some users from team",
+	Example:    "  team remove myteam user@example.com username",
+	Deprecated: "please use \"archive\" instead",
+	Args:       cobra.MinimumNArgs(2),
+	RunE:       withClient(archiveUsersCmdF),
+}
+
+var TeamUsersArchiveCmd = &cobra.Command{
+	Use:     "archive [team] [users]",
+	Short:   "Archive users from team",
+	Long:    "Archive some users from team",
+	Example: "  team archive myteam user@example.com username",
+	Args:    cobra.MinimumNArgs(2),
+	RunE:    withClient(archiveUsersCmdF),
```

Comment:
We should modify the function names as well:

```suggestion
	RunE:    withClient(teamUsersArchiveCmdF),
```